website/docs/components/form/super-select/partials/code/component-api.md

```diff
@@ -13,7 +13,12 @@ While the [ember-power-select documentation](https://ember-power-select.com/docs
 
 !!! Info
 
-The default values of some [ember-power-select](https://ember-power-select.com/docs/api-reference) options have been overridden in some cases to either resolve accessibility issues or to customize according to our needs. These options are documented below.
+The default values of some [ember-power-select](https://ember-power-select.com/docs/api-reference) options have been overridden to either resolve accessibility issues or to customize according to our needs.
+
+Overrides to fix accessibility issues:
+
+- `renderInPlace`: Always set as `true`. (The opener element or “trigger” should be next to dropdown content it hides & reveals.)
```

Comment:
Super small nit, but i'd remove the parenthesis on this line and the following line.
```suggestion
- `renderInPlace`: Always set as `true`. The opener element or “trigger” must be next to dropdown content it hides & reveals.
```